falcon/app.py

```diff
@@ -43,10 +44,10 @@
 ])
 
 
-class API:
```

Comment:
I believe we also may want to leave support for `API` as a compatibility alias for at least one Falcon release, even if 3.0 is going to be a breaking one.
Add let's add/leave some unit tests verifying the `API` alias still works.

Reply:
@vytas7 I Agree, but which tests we need to keep for the API alias? to make sure that all changes are safe

Reply:
You could probably `grep` the Falcon codebase (which I see you have already been doing extensively :+1: ) to see if `API` is used indirectly in any funky way, and add coverage for that.

Otherwise use your best judgement. To give an idea, maybe you could take grab a couple of tests exercising `API` / `App` creation (those constructing `TestClient` should do), and parametrize them to check both classes.

Reply:
Thanks, will do!

Reply:
Ok, added temporary `test_alias` file, and two tests for it. I think it will be enough for the alias.